frontend/src/tests/import-export.test.js

```diff
@@ -44,6 +55,89 @@ describe("Import/export library functionality", () => {
         },
     ];
 
+    /**
+     *  Create a `File` object containing of the specified format.
+     * This function is a copy of function dataToFile in ../libs/importExportUtils.
+     * The only difference is this function returns a FileAPI File object but not a File object.
+     * The reason is Jest is running on NodeJS which does not support File object.
+     * By calling this function only for testing, the app's export functionality will not be affected,
+     * while the test can still test the export functionality.
```

Comment:
Can't you do `global.File = FileAPI.File` and then use the original function?

Reply:
the file-api I used has different constructor than the original File API, and it requires a name field to exist thus conflicts with the original File API's constructor.

Reply:
Can you create a shim that imitates the native file object? I really don't like this duplicated code.